prov/sm2/src/sm2.h

```diff
@@ -95,6 +95,9 @@ enum {
 	sm2_proto_max,
 };
 
+/* Protocol flags */
+#define SM2_ATOMIC_RMA_REQ (1 << 1)
```

Comment:
I would suggest making this as generic as possible to make it more usable if needed in the future so I would change this to SM2_RMA_REQ in case RMA could use it

Reply:
@lrbison are you using such a flag in RMA? Are you OK with calling it `SM2_RMA_REQ`?